pkg/servicescm/servicescm.go

```diff
@@ -127,6 +135,32 @@ func GetLatest(c client.Client, ctx context.Context, namespace string) (*core.Co
 	return &servicesCMs[0], nil
 }
 
+// GetVersionCM returns the configMap associated with the node's version annotation
```

Comment:
The function is being introduced in this commit but being used in a follow up commit. It should be named `GetVersion()` to avoid stuttering given the call sites will read `servicescm.GetVersion()`.

You don't need to be constrained by the commit prefixes when introducing changes. Introducing them in a logical manner is more important as it makes reviews easier.

Reply:
agreed, apologies I didn't double-check and this was added in a local rebase conflict.

Reply:
I don't think this function should be added to this package, this package contains general functions for working with the Windows service CMs, IMO the logic introduced in this function is a bit too specialized, due to its use of `node.Annotations[metadata.VersionAnnotation]`. This should 

I don't see it being used anywhere but within cleanup.go, and would fit better there as a helper function.

Reply:
@sebsoto we are giving conflicting reviews 😄  I asked for it to be moved here given it has to do with the services ConfigMap. I don't see why it can't be added here as a helper. One alternative would be to make the `VersionAnnotation` an input.